CMakeLists.txt

```diff
@@ -229,13 +229,20 @@ endif()
 # We need the flag on riscv, armv6 and m68k.
 include(CheckCXXSourceCompiles)
 check_cxx_source_compiles("#include <cstdint>
+#include <atomic>
 int main() {
   uint64_t x = 1;
   __atomic_add_fetch(&x, 0, __ATOMIC_RELAXED);
```

Comment:
Now you can remove these lines and `#include <cstdint>`?

Reply:
Sure.